Add one-liner so the students' eyes don't glaze over (#40)
tekknolagi/isdt: index.md

## Overview

> Learn the tools you are expected to know as a working software engineer that
> make you more efficient at writing better code and tackle problems you
> previously thought impossible.

Reviewer: ```suggestion
Learn tools you'll be expected to know as a working software engineer, which will
help you write better code, collaborate with others, and tackle problems you
previously thought impossible.
```
